wweval/R/format_data_for_stan.R

```diff
@@ -2,7 +2,8 @@
 #'
 #' @param model_type string indicating which model we are getting data for
 #' Options are `ww` or `hosp`
-#' @param forecast_date string indicating the forecast date
+#' @param forecast_date string indicating the forecast date in ISO8 convention
```

Comment:
```suggestion
#' @param forecast_date string indicating the forecast date in ISO 8601 convention
```

I think? https://en.wikipedia.org/wiki/ISO_8601

Reply:
Good catch, probably need to fix this in the package code